### ISSUE TITLE
[wg/webauthn] Web Authentication Working Group

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#446]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-drafts/2024/wg-webauthn.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F04%2Fwebauthn-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2024%2Fwg-webauthn.html)

[diff with charter template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2024%2Fwg-webauthn.html)

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

None. The Working Group is the right place.

Known or potential areas of concern: 

None.

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

https://github.com/w3c/charter-drafts/issues

Anything else we should think about as we review? 

Nope

cc @nadalin @YubicoDemo

**Comment by plehegar** [comment]
Discussed by WG at:
* [02/07](https://www.w3.org/2024/02/07-webauthn-minutes.html#t00)
* [02/21/](https://www.w3.org/2024/02/21-webauthn-minutes.html)

**Comment by ruoxiran** [comment]
no comments or requests from APA.

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by plehegar** [comment]
All good for PING

**Comment by plehegar** [comment]
(TiLT approval has been requested, with a month extension)

**Comment by himorin** [comment]
- charter status at the top table is better to be replaced with standard two links??
- in 5.2, FIDO 2.0 WG lacks a link, and also it's [FIDO2 Technical Working Group](https://fidoalliance.org/members/working-groups/#fido2-technical-working-group)? (not sure 'Technical' is mandatory or not)

**Comment by svgeesus** [comment]
Why has the charter template text about testing been removed?

**Comment by simoneonofri** [comment]
From the Security side and as mentioned in the call, I would propose in the Success Criteria to add the following wording:

> The security considerations section must include a comprehensive threat model with threats, attacks, mitigations and residual risks.

This structure of this section is indicated by the [Security and Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/#considerations) and in the referenced [RFC3552 - Guidelines for Writing RFC Text on Security Considerations](https://www.rfc-editor.org/rfc/rfc3552).

In this specific case, the section has very good content and refers to the Web Authentication Threat Model within the FIDO documentation.

**Comment by plehegar** [comment]
@himorin @svgeesus @simoneonofri , I addressed all of the comments as part of https://github.com/w3c/charter-drafts/pull/491

**Comment by plehegar** [comment]
changes in https://github.com/w3c/charter-drafts/pull/491 were approved by the WG btw

**Comment by svgeesus** [comment]
Looks good to me now, thanks!

**Comment by nadalin** [comment]
Thank you

**Comment by simoneonofri** [comment]
thank you @plehegar

**Comment by plehegar** [comment]
Announced
https://lists.w3.org/Archives/Member/w3c-ac-members/2024AprJun/0017.html
